src/Fancourier/Request/GetCountiesExternal.php

```diff
@@ -42,7 +42,7 @@ public function getCountry()
      */
     public function setCountry($county)
     {
-        $this->county = $county;
+        $this->country = $county;
         return $this;
     }
```

Comment:
Aici defapt trebuia modificat `county` cu `country` in toate cele 3 instante. Inclusiv parametrul functiei si assign-ul.